lib/debugger/formatTable.luau

```diff
@@ -1,3 +1,39 @@
+local stringColor = "#dcdcaa"
+local numberColor = "#b0c4de"
+local keywordColor = "#c586c0"
+
+function colorize(value, color)
+	if type(value) == "number" then
+		value = string.format("%.1f", value)
+	end
+
+	return '<font color="' .. color .. '">' .. value .. "</font>"
+end
+
+function colorizeMany(color, ...)
```

Comment:
This is redundant in handling variadics, You know there can only be max 3 values, i.e. Color3, and it would never handle more. The exception to that would be if you wanted to deal with colour sequences, but then you would write a different function anyways. 

```luau
function colorizeMany(color, x, y, z) 
    return colorize(color, x), colorize(color, y), colorize(color, z)
end
```

Reply:
Not true-UDim2 has 4 number components!

Reply:
Ah fair point, I completely forgot about UDim2... Id be okay with extending it with `colorize(color, w)` but this is no longer an issue to me. Good example